tests/brain/test_enum.py

```diff
@@ -493,3 +493,32 @@ def pear(self):
         for node in (attribute_nodes[1], name_nodes[1]):
             with pytest.raises(InferenceError):
                 node.inferred()
+
+    def test_enum_members_uppercase_only(self) -> None:
+        """Originally reported in https://github.com/pylint-dev/pylint/issues/7402.
+        ``nodes.AnnAssign`` nodes with no assigned values do not appear inside ``__members__``.
+
+        Test that only enum member `MARS` appears in the `__members__` container while
+        attributes `radius` and `mass` do not.
+        """
+        enum_class, enum_member = astroid.extract_node(
+            """
+        from enum import Enum
+        class Planet(Enum): #@
+            MARS = (1, 2) #@
+            radius: int = 1
+            mass: int
+
+            def __init__(self, mass, radius):
+                self.mass = mass
+                self.radius = radius
+
+        Planet.MARS.value
+        """
+        )
+        enum_class.infer()
```

Comment:
```suggestion
```

Reply:
Thanks! I won't commit this directly because there is an unused variable to remove also.